tests/solvers/quadratic/fgw_test.py

```diff
@@ -268,7 +268,7 @@ def test_fgw_lr_generic_cost_matrix(
     lr_prob = prob.to_low_rank()
     assert lr_prob.is_low_rank
 
-    solver = gw_solver.GromovWasserstein(rank=5, epsilon=1.0)
+    solver = gw_solver.GromovWasserstein(rank=5, epsilon=10.0)
```

Comment:
do you still need to increase epsilon with the changed dims?

Reply:
I checked again, and for this specific seed yes, we need to increase epsilon, but for some others it's not necessary...

Reply:
sounds like a fairly steep change :) 

wouldn't it be more reasonable to change the tolerance? What kind of error is `epsilon=1.0` triggering?

Reply:
the algorithm doesn't converge (`assert out.converged` returns an `AssertionError`). I've just opened an issue to report this as a bug (#302).